src/main/java/com/github/wovnio/wovnjava/Settings.java

```diff
@@ -273,7 +270,7 @@ String hash() throws NoSuchAlgorithmException {
         for (String q : query) {
             md.update(q.getBytes());
         }
-        md.update(sitePrefixPathWithSlash.getBytes());
```

Comment:
I do not know much about this code.
However, I think that md will be changed if slash is removed.
Is this OK?

Reply:
Good point. Yes, `md` will be slightly different here. It is used to create the settings hash that is part of the cache key for translation API requests. This should be ok because the cache key will be different for a new version of wovnjava anyways (the version number is included in the cache key).